src/pixie/paths.nim

```diff
@@ -886,6 +889,7 @@ proc fillShapes(
   image: Image,
   shapes: seq[seq[Vec2]],
   color: ColorRGBA,
+  blendMode: BlendMode,
```

Comment:
I think winding rule is more common then blend mode. Maybe just put winding rule first, then blend Mode.